app/Http/Controllers/CustomerController.php

```diff
@@ -70,8 +69,8 @@ public function import(Request $request)
                         'password' => Hash::make($row[10]),
                     ]);
                     Customer::create([
```

Comment:
```suggestion
                    Customer::create([
```
ekhane repo call koro and create repo te koro